packages/color-kit-block/src/Palette.tsx

```diff
@@ -1,17 +1,30 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { Text } from '@frontify/fondue';
+import { Text, merge } from '@frontify/fondue';
 
-import type { PaletteProps } from './types';
+import { EmptyView } from './EmptyView';
 import { Color } from './Color';
+import type { PaletteProps } from './types';
 
 export const Palette = ({ palette, isEditing }: PaletteProps) => {
     const { name, colors = [] } = palette;
 
+    if (colors.length === 0) {
+        return <EmptyView paletteName={`${name || 'Unitled Palette'} (no colors added)`} />;
```

Comment:
```suggestion
        return <EmptyView paletteName={`${name ?? 'Untitled Palette'} (no colors added)`} />;
```

Reply:
The idea was to have `||` because when the name is an empty string, we have `Untitled Palette` as a title. With `??` we would just get an empty string and a wrong palette name.

Reply:
true that . keep || 👍